### DISCUSSION TITLE
How to use a loader that is not included in dist.min.js ?

### DISCUSSION THREAD

**Discussion by bobfortin64** [visgl/deck.gl#8276] [General]
I am using straight js without react by using <script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>

I want to load a PointCloudLayer from a .las file using the [LASLoader] from loaders.gl.

Can someone help figure out how to do this? Is there a simple trick or do I have to build deck.gl and somehow get the loaders i need included in a custom dist.min.js?

-Bob

**Comment by Pessimistress** [comment]
You can include `https://unpkg.com/@loaders.g/las@^3.0.0/dist/dist.min.js`

Then access the global `loaders` object:

```js
const {LASLoader} = loaders
```

**Reply by bobfortin64** [reply]
Thank you! I was able to get it to work.

**Reply by bobfortin64** [reply]
Do you know if it is possible to make a custom build of deck.gl to include several loaders into a custom dist.min.js?

-Bob
